client/rpc-spec-v2/src/chain_spec/api.rs

```diff
@@ -0,0 +1,53 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2017-2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! API trait of the chain spec.
+
+use jsonrpsee::{core::RpcResult, proc_macros::rpc};
+use sc_chain_spec::Properties;
+
+#[rpc(client, server)]
+pub trait ChainSpecApi {
+	/// Get the chain name, as present in the chain specification.
+	///
+	/// # Unstable
+	///
+	/// This method is unstable and subject to change in the future.
```

Comment:
why?

Reply:
Although they will be part of the final RPC, these methods are expected to go through a testing phase before standardizing and replacing the old API. During this time (1-2months TBD), we can still add breaking changes. At least, that's how I interpret the `unstable` from the [spec](https://paritytech.github.io/json-rpc-interface-spec/grouping-functions-and-node-capabilities.html)